edflow/util/edexplore.py

```diff
@@ -35,11 +35,30 @@ def isflow(obj):
     Returns
     -------
     bool
-        Returns True for three-dimensional numpy arrays where third dimension is two.
+        Returns True for rank-three numpy arrays where third axis is two.
     """
     return isinstance(obj, np.ndarray) and len(obj.shape) == 3 and obj.shape[2] in [2]
 
 
+def issegmentation(obj):
+    """Heuristic if item could be displayed as a segmentation mask
+
+    Parameters
+    ----------
+    obj : Any
+        item
+
+    Returns
+    -------
+    bool
+        Returns True for rank-three numpy arrays with values boolean or in
+        [0,1]
+    """
+    is_ = isinstance(obj, np.ndarray) and len(obj.shape) == 3
+    is_ = is_ and ((obj.dtype == np.bool) or (0.0 <= obj.min() and obj.max() <= 1.0))
```

Comment:
Are we sure, this is not a dangerous heuristic? Images can also fit that heuristic.
As one can choose which display method to use in edexplore that is no problem there, but if someone would use the heuristic elsewhere it might introduce unexpected behavior. Maybe add a warning in the documentation.

Reply:
good idea! done!